Esteban-Salas-Posada/tarea-13-proyecto/project/src/App.js

```diff
@@ -0,0 +1,90 @@
+import React, { Component } from 'react'
+import Seeker from './components/Seeker'
+import Result from './components/Result'
+
+class App extends Component{
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      word: '',
+      image: [],
+      page: ''
+    };
+  }
+
+  // Al cambiar de pagina hacer scroll hacia el inicio
+  scroll = ()=>{
+    const element = document.querySelector('.jumbotron')
+    element.scrollIntoView('smooth', 'start')
+  }
+
+  // Ir a la anterior pagina de imagenes
+  previousPage = ()=>{
+    let page = this.state.page
+    if(page===1 ) return null
+    page -= 1
+    this.setState({
+      page: page
+    }, /* Esto es un callback, se usa porque tenemos que mandar llamar la funcion para que re-renderice pues el cambia el estado pero no renderiza */()=>{
+      this.consultAPI()
+      this.scroll()
+    })
+  }
+
+  // Ir a la siguiente pagina de imagenes
+  nextPage = ()=>{
+    let page = this.state.page
+    page += 1
+    this.setState({
+      page: page
+    }, /* Esto es un callback, se usa porque tenemos que mandar llamar la funcion para que re-renderice pues el cambia el estado pero no renderiza */ ()=>{
+      this.consultAPI()
+      this.scroll()
+    })
+  }
+
+  consultAPI = () =>{
+    const word = this.state.word
```

Comment:
@esteban-salas , even this is not bad, is a good practice to keep login and API call separated from components, please research about how to construct a service later with context and the API calls there.